tests/framework/test_time.py

```diff
@@ -24,13 +24,13 @@ def test_align_times(SimulationContext, base_config, components):
     sim = SimulationContext(base_config, components)
     sim.setup()
     sim.initialize_simulants()
-    pop_size = len(sim._population.get_population(True))
+    pop_size = len(sim.get_population())
```

Comment:
Why did you drop `True` here?

Reply:
`SimulationContext`'s get_population method defaults to True, as opposed to the population managers, which does not specify a default for `untracked`.